config.proto

```diff
@@ -17,18 +17,35 @@ message AgentConfig {
     // reporting holds the reporting settings for the agent
     Reporting reporting = 2;
 
+
     // dataCapture describes the data being captured by instrumentation
     DataCapture dataCapture = 3;
 }
 
 // Reporting covers the options related to the mechanics for sending data to the
 // tracing server o collector.
 message Reporting {
-    // address represents the host for reporting the traes e.g. api.traceable.ai
+    // address represents the host for reporting the traces e.g. api.traceable.ai
     google.protobuf.StringValue address = 1;
 
     // secure when false, permits connecting to the trace endpoint without a certificate
     google.protobuf.BoolValue secure = 2;
+
+    // user specific token to access Traceable API
+    google.protobuf.StringValue token = 3;
+
+    // opa describes the setting related to the Open Policy Agent
+    Opa opa = 4;
+}
+
+// Opa covers the options related to the mechanics for getting Open Policy Agent configuration file.
+// The client should use secure and token option from reporting settings.
+message Opa {
+    // address represents the host for polling OPA config file e.g. http://opa.traceableai:8181/
```

Comment:
I think this is confusing. In line https://github.com/hypertrace/agent-config/pull/23/files#diff-3eaf2c85e69e9ea4804954dd98f3b0e3de1047da36714aef929a41efac2fbd54R28 we specify address as `api.traceable.ai` because we expect the host and we decide what endpoint to use whereas in here we call it address but from the example I believe we expect an endpoint. Can we stick to the previous setting and use `address` and `secure` or instead move into `endpoint`? cc @pavolloffay @davexroth

Reply:
This is a fair point, and I can see how it could lead to confusion.  I think it's a good idea to rename to either `endpoint` or `url`